float32[6] pose_value
string unit
bool relative
---
bool success
string message
